src/common/util.js

```diff
@@ -786,6 +786,27 @@ function addTrailingComment(node, comment) {
   addCommentHelper(node, comment);
 }
 
+function isWithinParentArrayProperty(path, propertyName) {
+  const node = path.getValue();
+  const parent = path.getParentNode();
+
+  if (parent == null) {
+    return false;
+  }
+
+  if (
+    !(
+      parent[propertyName] &&
+      Object.prototype.toString.call(parent[propertyName]) === "[object Array]"
```

Comment:
You can use `Array.isArray` (we use it in several places)

Reply:
`Array.isArray` isn’t supported on Node 4; do we polyfill?

Reply:
@j-f1 it is supported. For node 4, it doesn't work with Proxys or subclasses